draft-ietf-ppm-dap.md

```diff
@@ -512,12 +513,42 @@ implies HTTP status code 400 Bad Request unless explicitly specified otherwise.
 
 # Protocol Definition
 
-DAP has three major interactions which need to be defined:
+DAP is structured as an HTTP application consisting of three major interactions
+or sub-protocols:
 
-* Uploading reports from the client to the aggregators, specified in
-  {{upload-flow}}
-* Computing the results of a given measurement, specified in {{aggregate-flow}}
-* Collecting aggregated results, specified in {{collect-flow}}
+* Uploading reports from the client to the aggregators, using the HPKE
+  configuration ({{hpke-config-resource}}) and report ({{report-resource}})
+  resources, as described in {{upload-flow}}.
+* Computing the aggregate over sets of reports, using the aggregation job
```

Comment:
I think this would be a bit confusing, because in this bullet, we are discussing the aggregation sub-protocol, which works in terms of aggregation jobs, and those aren't necessarily equivalent to batches.